### PR TITLE
recovery counters

### PR DESCRIPTION
**Pull request by dachary**

### CODE REVIEW

**Comment by dachary** [comment]
@athanatos in the case of ECBackend, where is it more convenient to retrieve the pool size / data chunk count ? Also I'm unsure where to find the list of omap retrieved during recovery.

**Comment by dachary** [comment]
@athanatos ping ?

File: src/osd/ECBackend.cc [hunk 1]
```diff
@@ -599,7 +594,11 @@ void ECBackend::continue_recovery_op(
 		object_stat_sum_t());
 	    }
 	  }
-	  get_parent()->on_global_recover(op.hoid);
+	  object_stat_sum_t stat;
+	  stat.num_bytes_recovered = op.recovery_info.size;
+	  stat.num_keys_recovered = 0; // ??? op ... omap_entries.size(); ?
```
Comment (liewegas) [inline src/osd/ECBackend.cc]: 0 is fine, since we have no omap for EC pools, right?

Comment (dachary) [inline src/osd/ECBackend.cc]: Yes !

**Comment by athanatos** [comment]
Tested. http://pulpito.ceph.com/samuelj-2016-02-01_12:57:02-rados-wip-sam-testing-distro-basic-smithi/

LGTM.  I'd merge it now, but I'm not sure why the PR was closed.

**Comment by athanatos** [comment]
@dachary

**Comment by dachary** [comment]
@athanatos I closed it because I assumed you incorporated it somewhere else. No other reason.

**Pull request merged**